### PR TITLE
LDP 6757 Second pr 

### PR DESCRIPTION
**Pull request by alvarosaugarlr**
Second pr of [LPD-6757](https://liferay.atlassian.net/browse/LPD-6757)

First pr - https://github.com/liferay-appsec/liferay-portal/pull/1977

### CODE REVIEW

File: modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java [hunk 1]
```diff
@@ -191,11 +201,32 @@ private void _sendNotification(long companyId) {
...
 
-						if ((daysBetween == MONTH) ||
-							(daysBetween == TEN_DAYS) || (daysBetween == DAY)) {
+						int compareDaysBetweenExpiration = DateUtil.compareTo(accessTokenExpirationDate, new Date());
+
+						//get expando
+
+				//		if ((daysBetween == MONTH) ||
+				//			(daysBetween == TEN_DAYS) || (daysBetween == DAY)) {
+
```
Comment (stian-sigvartsen) [inline modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java:216]: @alvarosaugarlr can you remove these comments from the commit?

File: modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java [hunk 2]
```diff
@@ -254,8 +254,36 @@ private void _sendNotification(long companyId) {
...
+							subscriptionSender.setReplyToAddress("admin@liferay.com");
+
+							subscriptionSender.addRuntimeSubscribers("admin@liferay.com", "toName");
+
+							subscriptionSender.setMailId("popPortletPrefix", "ids");
+
+							subscriptionSender.flushNotifications();
+
+							//_sendNotificationEvent(users, body);
+							//_sendEmail(company, subject, body, users);
```
Comment (stian-sigvartsen) [inline modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java]: And here also

File: modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java [hunk 3]
```diff
@@ -254,8 +254,36 @@ private void _sendNotification(long companyId) {
...
+
+							String defaultEmailFromAddress = "scim-notification@" + company.getMx();
+							String defaultEmailFromName = "SCIM-Notification";
+
+							String mail = users.get(0).getEmailAddress();
+
+							SubscriptionSender subscriptionSender = new SubscriptionSender();
+
+							//subscriptionSender.setClassName();
+							//subscriptionSender.setClassPK();
```
Comment (stian-sigvartsen) [inline modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java]: Here also

File: modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java (hunk 2 above)
Comment (alvarosaugarlr) [reply modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java]: The comments are in previos commits but in the [final version](https://github.com/liferay-appsec/liferay-portal/blob/621ddc6205b430059cb616660b8af203fdfc6d32/modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java) is not commented.

File: modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java (hunk 3 above)
Comment (alvarosaugarlr) [reply modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java]: The comments are in previos commits but in the [final version](https://github.com/liferay-appsec/liferay-portal/blob/621ddc6205b430059cb616660b8af203fdfc6d32/modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java) is not commented.

**Comment by alvarosaugarlr** [comment]
ci:test:sf

**Comment by alvarosaugarlr** [comment]
ci:test:stable

**Comment by alvarosaugarlr** [comment]
ci:test:relevant

**Comment by alvarosaugarlr** [comment]
ci:test:security

**Comment by alvarosaugarlr** [comment]
ci:test:sf

**Comment by alvarosaugarlr** [comment]
ci:test:stable

**Comment by alvarosaugarlr** [comment]
ci:test:relevant

**Comment by alvarosaugarlr** [comment]
ci:test:security

**Comment by stian-sigvartsen** [comment]
Some commits squashed to https://github.com/liferay-appsec/liferay-portal/pull/1984

**Pull request closed without merging**